.test-infra/terraform/google-cloud-platform/google-kubernetes-engine/outputs.tf

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output kubernetes_api_endpoint {
+  value = google_container_cluster.default.endpoint
+}
+
+output cluster_ca_certificate {
```

Comment:
Thank you for adding outputs :-). Could you tell me what this output is needed for?

Reply:
This output is used in the upper module for helm to authenticate

https://github.com/apache/beam/pull/30847/files#diff-06d3b2b8f4c2307a7709a5e070cf31b505b71432cc46eca5871a402e74163eb8R33-R39

Reply:
I think the provisioning of the Kubernetes cluster and any workloads that depend on it should be in separate terraform modules. Then one would just follow typical gcloud command to connect.